InnoTopicWebsite/src/app/skills/work-experience.component.html

```diff
@@ -128,9 +116,9 @@ <h2 class="section-title">Experience <!--<span class="comment">(hover/tap to hig
   <!--    <app-skill-levels-icons [skillLevels]="{have: 'beginner', want: 'expert'}"></app-skill-levels-icons>-->
   <!--    <app-skill-levels-icons [skillLevels]="{have: 'advanced'}"></app-skill-levels-icons>-->
       <p class="example">Example:</p>
-      <app-topic-skill [tId]="'Python'"
+      <app-topic-skill [tId]="'Angular'"
```

Comment:
@samyakjainnnnn Karol says it should stay python - do you know if it was a manua update or a merge conflict resolution issue?